content/en/blog/2022/how-to-get-started-with-opentelemetry-go.md

```diff
@@ -0,0 +1,420 @@
+---
+title: Go Web App Instrumentation
+linkTitle: Go App Instrumentation
+date: 2022-06-27
+author: Naveh Mevorach (Aspecto)
+canonical_url: https://www.aspecto.io/blog/opentelemetry-go-getting-started/
+---
+
+
+In this blog post, you will learn hands-on how to create and visualize traces with OpenTelemetry Go without prior knowledge.
+
+We will start with creating a simple to-do app that uses Mongo and the Gin framework. Then, we will send tracing data to Jaeger Tracing for visualization. You can find all the relevant files in this [Github repository](https://github.com/aspecto-io/opentelemetry-examples/tree/master/go).
+
+!["OpenTelemetry Go The Mandalorian"](https://www.aspecto.io/wp-content/uploads/2022/06/OpenTelemetry-Go-The-Mandalorian-2048x1406.png)
+
+## What to Expect
+
+- Intro to OpenTelemetry
+- Hello world: OpenTelemetry GO example
+  - Create main.go file with Gin and Mongo
+  - Install OpenTelemetry GO
+  - Gin instrumentation: gin.Context
+- Visualization with Jaeger
```

Comment:
This "What to expect" section seems to essentially be a Table of Contents. I'd drop it given that a real ToC with hyperlinks is autogenerated. WDYT?